src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -2894,11 +2894,13 @@ public function get_comment_type(): ?string {
 	 * @return string
 	 */
 	public function get_modifiable_text(): string {
-		if ( null === $this->text_starts_at || 0 === $this->text_length ) {
+		$has_enqueued_update = isset( $this->lexical_updates['modifiable text'] );
+
+		if ( ! $has_enqueued_update && ( null === $this->text_starts_at || 0 === $this->text_length ) ) {
```

Comment:
I don't know enough about the internals to say anything intelligent here, but out of curiosity, and to confirm to my brain, we want to  prevent unnecessary early returns when `$this->lexical_updates['modifiable text']` is set here (?)

Reply:
this is right, because when we started with an empty modifiable text (which is only possible in "self-contained" elements, because an empty text node doesn't exist) we were returning early even if enqueued updates were applied.